### ISSUE TITLE
pachctl put file EOF on large rosbag file

### ISSUE THREAD

**Issue by davidthewatson** [pachyderm/pachyderm#3959]
I have pachyderm 1.9.2 running on minikube. I'm trying to put a file such as:

    [watson@watson-xps13 rosbags]$ ls -lr *.bag
    -rwxr-xr-x 1 watson watson 899736722 Jul 31 15:56 2019-04-30-06-35-14_0.bag

with:
```
[watson@watson-xps13 rosbags]$ pachctl put file -v rosbags@master -f 2019-04-30-06-35-14_0.bag 
[0000]  INFO parsed scheme: "" source=etcd/grpc
[0000]  INFO scheme "" not registered, fallback to default scheme source=etcd/grpc
[0000]  INFO ccResolverWrapper: sending update to cc: {[{192.168.39.15:30650 0  <nil>}] } source=etcd/grpc
[0000]  INFO ClientConn switching balancer to "pick_first" source=etcd/grpc
[0000]  INFO pickfirstBalancer: HandleSubConnStateChange: 0xc000212e90, CONNECTING source=etcd/grpc
[0000]  INFO pickfirstBalancer: HandleSubConnStateChange: 0xc000212e90, READY source=etcd/grpc
EOF
[watson@watson-xps13 rosbags]$ pachctl list repo
NAME       CREATED        SIZE (MASTER) 
rosbags    46 minutes ago 0B            
[watson@watson-xps13 rosbags]$ pachctl list file rosbags@master
/pachyderm_pfs/branches/rosbags/ master not found
```

But I keep getting 0 bytes in the rosbags repo. the `pachctl put file` documentation didn't seem to list a maximu file size. Is it possible this is too big for the lightweight storage approach in minikube?

**Comment by JoeyZwicker** [comment]
@davidthewatson Are other put file operations working normally? As in, you can add a local text file and it all works smoothly?

I've added multi-GB files into my minikube setup before and it seems to work fine so I wonder if it could be something strange with that file format? Can you test putting a similarly sized file of a different type and see if that's an issue?

**Comment by davidthewatson** [comment]
Yes, all the tutorials are working but they are comparatively tiny. I will test large txt file in the morning and report my findings.

**Comment by jdoliner** [comment]
@davidthewatson sorry for not following up on this, were you able to test this with a large, non-bag file? I'd be surprised if this has anything to do with the specifics of the file format, but it's always possible. Another important question here is how are you connecting to the pachd, and could we see the logs from during the put file?

**Comment by JoeyZwicker** [comment]
@davidthewatson just wanted to check in again to see if you have any more info you can provide on this. thanks

**Comment by pappasilenus** [comment]
Agent comment from John Karabaic in <a href='https://pachyderm.zendesk.com/agent/tickets/55'>Zendesk ticket #55</a>:<br><br>You can post your reproduction steps to this ticket.

**Comment by pappasilenus** [comment]
Agent comment from John Karabaic in <a href='https://pachyderm.zendesk.com/agent/tickets/55'>Zendesk ticket #55</a>:<br><br>Hi, Vincent,

Are you able to confirm that memory profile information?
